base/libgit2/types.jl

```diff
@@ -554,7 +562,10 @@ function with_warn{T}(f::Function, ::Type{T}, args...)
     end
 end
 
-
+"""
+Convert between the julia Type of a git object and
+the constant integer id code for that object type.
+"""
```

Comment:
Since the function signatures are given in the other places it might be nice to provide it here as well. (Likewise below)